Formula/bumpversion.rb

```diff
@@ -0,0 +1,20 @@
+class Bumpversion < Formula
+  include Language::Python::Virtualenv
+
+  desc "Increase version numbers with SemVer terms"
+  homepage "https://pypi.python.org/pypi/bumpversion"
+  url "https://github.com/peritus/bumpversion/archive/v0.5.3.tar.gz"
+  sha256 "97ac6efca7544853309b68efe92f113ab6bddb77ecbaefa5702a6183a30bcb33"
+
+  bottle :unneeded
+
+  depends_on "python"
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    assert_includes shell_output("script -q /dev/null #{bin}/bumpversion --help"), "bumpversion: v#{version}"
```

Comment:
Congratulations on your first submission to Homebrew @christopherdcunha ! 
To make the test more useful its better to write a test that tests the functionality of the software rather than just checking an output version string. Is that something you could write ?

Reply:
Alrighty @javian, I think this is ready to be re-reviewed

Reply:
Looks great! The only thing I'm pondering now as with the jinja formula is that I'm still doubting whether this shouldn't be bottled.

Reply:
@javian Just in case you didn't notice the change/force-push, I removed the `bottle :unneeded` declaration from this PR.